AllReadyApp/Web-App/AllReady/Views/Shared/_Layout.cshtml

```diff
@@ -129,6 +130,10 @@
         <script src="//d3js.org/d3.v3.min.js"
                 asp-fallback-src="~/lib/d3/d3.js">
         </script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.maskedinput/1.4.1/jquery.maskedinput.min.js"
+                asp-fallback-src="~/lib/jquery.maskedinput/dist/jquery.maskedinput.js"
+                asp-fallback-test="window.mask">

```

Comment:
Fallback test should be `window.jQuery && window.jQuery.fn && window.jQuery.fn.mask`. Loading `jquery.maskedinput` doesn't do anything to  `window.mask`.